src/components/tooltip/tooltip.tsx

```diff
@@ -129,17 +362,20 @@ export class Tooltip {
   /** Updates the position of the component. */
   @Method()
   async reposition(): Promise<void> {
-    const { popper, el, placement } = this;
+    const { popper, el, open, placement } = this;
+
+    if (!open || !popper) {
```

Comment:
We need to position/create popper before we open so that we can do the animation. I'll revert this part.